packages/client/src/chat.ts

```diff
@@ -589,16 +595,21 @@ export function add(data: ChatMessage, fast: boolean): void {
 }
 
 // addSelf is used when the client needs to send a chat message to itself
-export function addSelf(msg: string, room: string): void {
+export function addSelf(
+  msg: string,
+  room: string,
+  type: SelfChatMessageType = SelfChatMessageType.normal,
+): void {
+  const message = getChatMessage(msg, type);
   add(
     {
-      msg,
-      who: "",
+      msg: message,
+      who: "Hanab Live",
```

Comment:
my bet is that this string is a constant that lives somewhere else, and if it is, then it shouldn't be hardcoded here

Reply:
it's unreachable (`server_old/src/...` and `constants.go`), putting it into /data